File: app/src/main/java/com/olair/picsart/camera/utils/CameraSchedulers.java
package com.olair.picsart.camera.utils;

import io.reactivex.rxjava3.core.Scheduler;
import io.reactivex.rxjava3.schedulers.Schedulers;

/**
 * Create by oLair on 2020/11/4.
 */
public class CameraSchedulers {
    private final static Scheduler backgroundScheduler = Schedulers.newThread();

    public static Scheduler background() {
        return backgroundScheduler;
    }
}

File: app/src/main/java/com/olair/picsart/camera/utils/CameraSchedulers.kt
package com.olair.picsart.camera.utils

import io.reactivex.rxjava3.core.Scheduler
import io.reactivex.rxjava3.schedulers.Schedulers

/**
 * Create by oLair on 2020/11/4.
 */
object CameraSchedulers {
    private val backgroundScheduler = Schedulers.newThread()
    fun background(): Scheduler {
        return backgroundScheduler
    }
}